tools/ui

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [ "$1" == "run" ]; then
```

Comment:
Would be better to add this line at the beginning:
```sh
set -eu
```
so that it aborts its execution when one of the commands fails, and makes it an error to use undefined variables.